functions.php

```diff
@@ -77,6 +78,7 @@ function mp_ssv_enquire_scripts()
         } else {
             wp_enqueue_style('materialize', get_theme_root_uri() . '/ssv-material/css/' . get_current_blog_id() . '_materialize.css');
         }
+        wp_enqueue_style('font-awesome', get_theme_root_uri() .'/ssv-material/css/font-awesome.min.css');
```

Comment:
Where is this used, and why?

Reply:
I found where it is used and I would suggest to use the icons already provided by MaterializeCSS (implemented from Googles Material Icons).
Also the arrow right at the bottom of the posts at the post overview pages currently have 'padding-top: 3px;' and that is obviously not the correct way to center it. The correct way would be to set the 'line-height' to the same line-height as the 'READ MORE' text.